target/linux/kirkwood/patches-5.10/108-dns325.patch

```diff
@@ -0,0 +1,159 @@
+--- a/arch/arm/boot/dts/kirkwood-dnskw.dtsi
++++ b/arch/arm/boot/dts/kirkwood-dnskw.dtsi
+@@ -31,7 +31,7 @@
+ 		};
+ 	};
+ 
+-	gpio_fan {
++	fan0: gpio_fan {
+ 		/* Fan: ADDA AD045HB-G73 40mm 6000rpm@5v */
+ 		compatible = "gpio-fan";
+ 		pinctrl-0 = <&pmx_fan_high_speed &pmx_fan_low_speed>;
+@@ -189,33 +189,18 @@
+ 
+ 	partition@0 {
+ 		label = "u-boot";
+-		reg = <0x0000000 0x100000>;
++		reg = <0x0000000 0x0e0000>;
+ 		read-only;
+ 	};
+ 
+-	partition@100000 {
+-		label = "uImage";
+-		reg = <0x0100000 0x500000>;
+-	};
+-
+-	partition@600000 {
+-		label = "ramdisk";
+-		reg = <0x0600000 0x500000>;
+-	};
+-
+-	partition@b00000 {
+-		label = "image";
+-		reg = <0x0b00000 0x6600000>;
+-	};
+-
+-	partition@7100000 {
+-		label = "mini firmware";
+-		reg = <0x7100000 0xa00000>;
++	partition@e0000 {
++		label = "u-boot-env";
++		reg = <0x00e0000 0x020000>;
+ 	};
+ 
+-	partition@7b00000 {
+-		label = "config";
+-		reg = <0x7b00000 0x500000>;
++	partition@100000 {
++		label = "ubi";
++		reg = <0x0100000 0x7f00000>;
+ 	};
+ };
+ 
+--- a/arch/arm/boot/dts/kirkwood-dns325.dts
++++ b/arch/arm/boot/dts/kirkwood-dns325.dts
+@@ -2,11 +2,19 @@
+ /dts-v1/;
+ 
+ #include "kirkwood-dnskw.dtsi"
++#include <dt-bindings/thermal/thermal.h>
+ 
+ / {
+ 	model = "D-Link DNS-325 NAS (Rev A1)";
+ 	compatible = "dlink,dns-325-a1", "dlink,dns-325", "dlink,dns-kirkwood", "marvell,kirkwood-88f6281", "marvell,kirkwood";
+ 
++	aliases {
++		led-boot = &led_white_power;
++		led-failsafe = &led_white_power;
++		led-running = &led_white_power;
++		led-upgrade = &led_white_power;
++	};
++
+ 	memory {
+ 		device_type = "memory";
+ 		reg = <0x00000000 0x10000000>;
+@@ -24,24 +32,25 @@
+ 			     &pmx_led_white_usb>;
+ 		pinctrl-names = "default";
+ 
+-		white-power {
++		led_white_power: white-power {
+ 			label = "dns325:white:power";
+ 			gpios = <&gpio0 26 GPIO_ACTIVE_LOW>;
+ 			default-state = "keep";
+ 		};
+-		white-usb {
++		led_white_usb: white-usb {
```

Comment:
The led labels other than `led_white_power` are not used anywhere. Please drop them.